src/app/content/highlights/components/ColorPicker.tsx

```diff
@@ -139,6 +142,12 @@ const ColorPicker = ({className, ...props}: Props) => {
             ? props.onRemove ? props.onRemove() : null
             : props.onChange(style.label)}
       />)}
+      { props.size === 'small' ? null :
+        <TrashButton
+          size={props.size}
+          onClick={() => 'onRemove' in props && props.onRemove ? props.onRemove() : null}
```

Comment:
This is working great in the scenario we discussed. But should it also work before color selection, and after note creation? When I was testing those, nothing happened when I clicked the button. Maybe we just hide it if onRemove isn't set?

Reply:
I had to fiddle with useOnRemove so it would return null instead of a function that did nothing, but that makes it much less confusing to only have the trash can show up when it's functional.